src/Microsoft.ML.Mkl.Components/OlsLinearRegression.cs

```diff
@@ -36,12 +36,30 @@ namespace Microsoft.ML.Trainers
     /// ordinary least squares (OLS) for estimating the parameters of the linear regression model.
     /// </summary>
     /// <remarks>
-    /// <a href='https://en.wikipedia.org/wiki/Ordinary_least_squares'>Ordinary least squares (OLS)</a> is a parameterized regression method.
+    /// <format type="text/markdown"><![CDATA[
+    /// To create this trainer, use [Ols](xref:Microsoft.ML.MklComponentsCatalog.Ols(Microsoft.ML.RegressionCatalog.RegressionTrainers, System.String, System.String, System.String))
```

Comment:
>ners, System. [](start = 130, length = 13)

i'm not sure UID allows spaces. let's remove these and use the UID exactly as it comes from xref system to be on the safe side.